Makefile

```diff
@@ -290,6 +290,10 @@ benchmark:
 	@test -e bench-master.txt && benchstat bench-master.txt bench-$(REF_NAME).txt || benchstat bench-$(REF_NAME).txt
 .PHONY: benchmark
 
+test-mutation:
```

Comment:
I'm having a hard time understanding how this should be used. I've tried running `make test-mutation` and it outputs the following:

```
running mutation tests for the following module(s):
cannot find package "./x/commons/commons.go./x/profiles/abci.go" in:
        x/commons/commons.go./x/profiles/abci.go
```

Now, after this, nothing. I'm thinking it takes a while to execute, but it's not something that I'm warned about as a user when I run the command. I believe if that's the case, a warning should be printed

Reply:
Added a warning message and also made the processing messages printed